e2e-test/src/tests/delete.test.js

```diff
@@ -0,0 +1,66 @@
+import { vBtn } from 'jest-puppeteer-vuetify';
+import { CLIENT_URL } from '../util';
+
+
+describe('delete dataset', () => {
+  it('upload one dataset and then delete it', async () => {
+    await Promise.all([
+      page.goto(CLIENT_URL),
+      page.waitForNavigation({ waitUntil: 'networkidle0' }),
+    ]);
+    await expect(page).toClickXPath(vBtn('Your Datasets'));
+
+    const fileInputElementHandle = await page.waitForXPath("//input[@type='file']"); // $0.value == "" is true
+    await fileInputElementHandle.uploadFile('../e2e-test/data/Cancer chemo muscle (NMR).csv');
+    await page.waitForXPath("//div[@class='v-list__tile__title'][contains(text(),'Cancer_chemo_muscle_NMR.csv ')]");
+    await page.waitForXPath("//div[@class='v-list__tile__title'][contains(text(),'Cancer_chemo_muscle_NMR.csv ')]//following::span[contains(text(),'Dataset processed with 1 validation failures')]");
+
+    // click on 'X'
+    await page.waitForXPath("//div[@class='v-btn__content']//following::i[@class='v-icon mdi mdi-close theme--light']");
+    await expect(page).toClickXPath(vBtn("//i[@class='v-icon mdi mdi-close theme--light']"));
```

Comment:
Firstly, these two XPaths are practically the same. The first is slightly more explicit, while the second uses the XPath helper some, but they should generate very similar XPaths.
Secondly, this should work the same and be much cleaner:
```
vBtn(vIcon('mdi-close'))
```